### ISSUE TITLE
go get broken on this repro

### ISSUE THREAD

**Issue by morriswinkler-simple** [ericlagergren/decimal#86]
```
# github.com/ericlagergren/decimal/internal/arith
../../../../ericlagergren/decimal/internal/arith/intlen.go:48:6: x.CmpAbs undefined (type *big.Int has no field or method CmpAbs)

```

**Comment by ericlagergren** [comment]
Are you using Go 1.10?

**Comment by morriswinkler-simple** [comment]
```
go version go1.9.2 darwin/amd64
```

**Comment by ericlagergren** [comment]
Unfortunately, I only can support the most recent Go version (currently 1.10). For a while I tried supporting previous versions, but it became too burdensome to both add new features and add shims for older versions.

This is similar to how the Go project does things: they only support the two most recent versions. So, right now they support Go 1.9 and Go 1.10. Since I'm one guy doing this in my spare time, I only support the single most recent version.

If you want to create a PR, I'll gladly add a branch for Go 1.9. The code has already been written, I removed it in this commit: https://github.com/ericlagergren/decimal/commit/1d99a4eed92f77b3c4c57ebc0c3da15ca5137946
